### PR TITLE
Ci rn 803xx

### PR DESCRIPTION
**Pull request by aimurphy**
Release notes and adjacent doc updates

- [Continuous Integration](https://64e8c042718b072cfe7fa226--harness-developer.netlify.app/release-notes/continuous-integration)
- [What's new](https://64e8c042718b072cfe7fa226--harness-developer.netlify.app/release-notes/whats-new)
- [Early access features](https://64e8c042718b072cfe7fa226--harness-developer.netlify.app/release-notes/early-access)
- Build and upload an artifact - combine duplicate tabs in [Build a Docker image without pushing](https://64e8c042718b072cfe7fa226--harness-developer.netlify.app/docs/continuous-integration/use-ci/build-and-upload-artifacts/build-and-upload-an-artifact#build-a-docker-image-without-pushing) and add new section [Set kaniko runtime flags](https://64e8c042718b072cfe7fa226--harness-developer.netlify.app/docs/continuous-integration/use-ci/build-and-upload-artifacts/build-and-upload-an-artifact#set-kaniko-runtime-flags).
- CI-8370 - GitHub App as primary auth for GH connectors. Main changes are in github app support [Configure the GitHub connector](https://64e8c042718b072cfe7fa226--harness-developer.netlify.app/docs/platform/Connectors/Code-Repositories/git-hub-app-support#configure-the-github-connector) section and github connector settings ref [Authentication](https://64e8c042718b072cfe7fa226--harness-developer.netlify.app/docs/platform/Connectors/Code-Repositories/ref-source-repo-provider/git-hub-connector-settings-reference#authentication) section.
- CI-7600, DOC-1593, DOC-2871, DOC-2256: New [CI environment variables reference](https://64e8c042718b072cfe7fa226--harness-developer.netlify.app/docs/continuous-integration/use-ci/optimize-and-more/ci-env-var) - DRONE_ vars have been commented out until further decisions are made about how to document them.
- Revise [Fixed values, runtime inputs, and expressions](https://64e8c042718b072cfe7fa226--harness-developer.netlify.app/docs/platform/References/runtime-inputs)c, fix sidebar ranking for that  section.
- Reorg [Built-in CI codebase variables reference](https://64e8c042718b072cfe7fa226--harness-developer.netlify.app/docs/continuous-integration/use-ci/codebase-configuration/built-in-cie-codebase-variables-reference)

### CODE REVIEW

File: release-notes/continuous-integration.md [hunk 1]
```diff
@@ -18,43 +18,57 @@ Review the notes below for details about recent changes to Harness Continuous In
 Harness deploys changes to Harness SaaS clusters on a progressive basis. This means that the features and fixes that these release notes describe may not be immediately available in your cluster. To identify the cluster that hosts your account, go to the **Account Overview** page.
 :::
 
-## Latest - August 10, 2023, version 5301
+## Latest: Version 54xx <!-- Aug 17 2023 -->
```
Comment (aimurphy) [inline release-notes/continuous-integration.md:22]:
```suggestion
## Latest: Version 54xx

<!-- Aug 17 2023 -->

```

File: docs/continuous-integration/use-ci/optimize-and-more/ci-env-var.md [hunk 2]
```diff
@@ -0,0 +1,468 @@
...
+
+```yaml
+        payloadConditions:
+          - key: <+trigger.payload.ref>
+            operator: StartsWith
+            value: refs/tags/
+```
+-->
+
+## Difference between DRONE_ and PLUGIN_ variables
```
Comment (aimurphy) [inline docs/continuous-integration/use-ci/optimize-and-more/ci-env-var.md:46]:
```suggestion
## Difference between DRONE\_ and PLUGIN\_ variables
```

File: docs/continuous-integration/use-ci/optimize-and-more/ci-env-var.md [hunk 3]
```diff
@@ -0,0 +1,468 @@
...
+`HARNESS_ORG_ID`: Your Harness organization ID. Equivalent to `<+org.identifier>`.
+`HARNESS_PIPELINE_ID`: A pipeline's identifier, usually based on the pipeline's name. Equivalent to `<+pipeline.identifier>`.
+`HARNESS_PROJECT_ID`: The Harness project ID. Equivalent to `<+project.identifier>`.
+`HARNESS_SECRETS_LIST`
+`HARNESS_STAGE_ID`: The identifier for a stage. Equivalent to `<+stage.identifier>`.
+`HARNESS_STO_SERVICE_ENDPOINT`: The URL for STO service.
+`HARNESS_STO_SERVICE_TOKEN`: Authentication for the STO service.
+`HARNESS_TI_SERVICE_ENDPOINT`: The URL for Test Intelligence service.
+`HARNESS_TI_SERVICE_TOKEN`: Authentication for the Test Intelligence service.
+`HARNESS_WORKSPACE`: Duplicate of [`DRONE_WORKSPACE`](#drone_workspace).
```
Comment (aimurphy) [inline docs/continuous-integration/use-ci/optimize-and-more/ci-env-var.md:424]:
```suggestion
* `DELEGATE_SERVICE_TOKEN`
* `DELEGATE_SERVICE_ENDPOINT`
* `DELEGATE_SERVICE_ID`
* `DRONE_NETRC_USERNAME`
* `DRONE_NETRC_PASSWORD`
* `HARNESS_ACCOUNT_ID`: Your Harness account ID. Equivalent to `<+account.identifier>`.
* `HARNESS_BUILD_ID`: The incremental build ID for pipeline runs. Equivalent to `<+pipeline.sequenceId>` and [`DRONE_BUILD_NUMBER`](#drone_build_number).
* `HARNESS_EXECUTION_ID`: A pipeline's immutable UUID. Equivalent to `<pipeline.executionId>`
* `HARNESS_LOG_PREFIX`
* `HARNESS_LOG_SERVICE_ENDPOINT`
* `HARNESS_LOG_SERVICE_TOKEN`
* `HARNESS_ORG_ID`: Your Harness organization ID. Equivalent to `<+org.identifier>`.
* `HARNESS_PIPELINE_ID`: A pipeline's identifier, usually based on the pipeline's name. Equivalent to `<+pipeline.identifier>`.
* `HARNESS_PROJECT_ID`: The Harness project ID. Equivalent to `<+project.identifier>`.
* `HARNESS_SECRETS_LIST`
* `HARNESS_STAGE_ID`: The identifier for a stage. Equivalent to `<+stage.identifier>`.
* `HARNESS_STO_SERVICE_ENDPOINT`: The URL for STO service.
* `HARNESS_STO_SERVICE_TOKEN`: Authentication for the STO service.
* `HARNESS_TI_SERVICE_ENDPOINT`: The URL for Test Intelligence service.
* `HARNESS_TI_SERVICE_TOKEN`: Authentication for the Test Intelligence service.
* `HARNESS_WORKSPACE`: Duplicate of [`DRONE_WORKSPACE`](#drone_workspace).
```

File: docs/continuous-integration/use-ci/codebase-configuration/built-in-cie-codebase-variables-reference.md [hunk 4]
```diff
@@ -8,292 +8,304 @@ helpdocs_is_private: false
...
-* `<+trigger.event>`: The trigger event type. For push triggers, it is `PUSH`.
-* `<+trigger.type>`: The trigger type. For webhook triggers, it is `Webhook`.
+### codebase.shortCommitSha
+
+* Value: The short SHA (seven characters) of the build's [commit SHA](#commitsha).
+* Expression: `<+codebase.shortCommitSha>`
+
+## Git user data
+
+### gitUser
```
Comment (aimurphy) [inline docs/continuous-integration/use-ci/codebase-configuration/built-in-cie-codebase-variables-reference.md:270]:
```suggestion
### codebase.gitUser
```

File: docs/continuous-integration/use-ci/codebase-configuration/built-in-cie-codebase-variables-reference.md [hunk 5]
```diff
@@ -8,292 +8,304 @@ helpdocs_is_private: false
...
+
+* Value: User name of the Git account associated with the build. Can be `null` or masked in build logs.
+* Expression:
+   * Manual builds: `<+codebase.gitUser>`
+   * Webhook triggers: `<+codebase.gitUser>` or `<+trigger.gitUser>`
+
+### gitUserAvatar
```
Comment (aimurphy) [inline docs/continuous-integration/use-ci/codebase-configuration/built-in-cie-codebase-variables-reference.md:277]:
```suggestion
### codebase.gitUserAvatar
```

File: docs/continuous-integration/use-ci/codebase-configuration/built-in-cie-codebase-variables-reference.md [hunk 6]
```diff
@@ -8,292 +8,304 @@ helpdocs_is_private: false
...
+
+* Value: Link to user avatar of the Git account associated with the build.
+* Expression: `<+codebase.gitUserAvatar>`
+
+### gitUserEmail
```
Comment (aimurphy) [inline docs/continuous-integration/use-ci/codebase-configuration/built-in-cie-codebase-variables-reference.md:282]:
```suggestion
### codebase.gitUserEmail
```

**Review by brian-f-harness — APPROVED** [review]
LGTM. Only made a few minor suggestions. You're aware of the merge conflicts. (I think one may be a file I updated yesterday.)

File: docs/continuous-integration/use-ci/optimize-and-more/ci-env-var.md [hunk 7]
```diff
@@ -0,0 +1,451 @@
...
+
+These variables are not supported because they are not applicable or incompatible with Harness CI.
+
+* `DRONE_SYSTEM_VERSION`: This variable is not applicable because it describes the Drone server version.
+* `DRONE_STAGE_VARIANT`: This variable is not supported because it is optional and only applies to ARM architectures.
+* `DRONE_STAGE_DEPENDS_ON`
+* `DRONE_BUILD_PARENT`: This variable is not applicable because it is for the Drone-specific *promotions* feature.
+* `DRONE_DEPLOY_TO`: This variable is not applicable because it is for continuous delivery in Drone.
+* `DRONE_ENV`
+* `DRONE_CALVER_SHORT`, `DRONE_CALVER_MAJOR_MINOR`, `DRONE_CALVER_MAJOR`, `DRONE_CALVER_MINOR`, `DRONE_CALVER_MICRO`, `DRONE_CALVER_MODIFIER`: Harness CI supports [`DRONE_CALVER`](#drone_calver) but not it's variations.
```
[the remark this answers is not in the document]
Comment (aimurphy) [reply docs/continuous-integration/use-ci/optimize-and-more/ci-env-var.md:451]: I don't know what they do and I think one of the unsupported ones doesn't exist anymore, but it was decided in tech review to include them anyways. I believe there are more variables that need to be added here, so hopefully this list will get more populated over time.

**Comment by rajatharanganath** [comment]
Can we keep the drone variables page hidden please. Since need other approvals and review from product on it.

**Comment by aimurphy** [comment]
> Can we keep the drone variables page hidden please. Since need other approvals and review from product on it.

If I hide it, then there is nothing to link to in the release note. If this is not ready to be documented, i don't think it should be in the release notes either.

**Pull request merged**